lib/thread/default/export/ecl/thread/utils.hpp

```diff
@@ -0,0 +1,27 @@
+//! \file
+//! \brief Default utilities.
+#ifndef LIB_THREAD_DEFAULT_UTILS_HPP_
+#define LIB_THREAD_DEFAULT_UTILS_HPP_
+
```

Comment:
just to clarify code style, should we use trailing underscore in define "guards?"